build-package.sh

```diff
@@ -28,6 +28,9 @@ case "$1" in
         ;;
     stage)
         craftctl default
+        if [ -e ${CRAFT_PROJECT_DIR}/test-debs ]; then
```

Comment:
Might be a good idea to put a comment above this.

Reply:
You are right. Done.